terraform/environments/mlra/ecs.tf

```diff
@@ -24,7 +24,7 @@ module "mlra-ecs" {
   app_count                 = local.application_data.accounts[local.environment].app_count
   ec2_ingress_rules         = local.ec2_ingress_rules
   ec2_egress_rules          = local.ec2_egress_rules
-  lb_tg_name                = local.application_data.accounts[local.environment].lb_tg_name
+  lb_tg_arn                 = module.alb.target_group_arn
   tags_common               = local.tags
```

Comment:
Implicit dependency now used to narrow down the actual link between the ALB and ECS module